src/Integration/Monolog/CloudWatch/src/CloudWatchLogsHandler.php

```diff
@@ -252,4 +262,39 @@ private function send(array $entries): void
 
         $this->client->putLogEvents($data);
     }
+
+    public function handle(LogRecord|array $record): bool
```

Comment:
Is it needed?

Reply:
@jderusse the idea was to handle both simple array and LogRecord support both simple

Reply:
If I'm not wrong, this method is already implemented in `AbstractProcessingHandler`

Which already handles `array` when Monolog 2 is installed and `LogRecord` when version 3 is installed.

I don't understand why we have to provide an implementation. Please tell me a case (little reproducer/gadget) to show me the issue.